style.css

```diff
@@ -624,155 +741,165 @@ input[type=submit]:hover {
 	height: 70px;
 	color: white;
 	grid-row: 7;
+	font-size: 16px;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 400;
 }
 
 .contact_footer {
 	bottom: 0;
 	position: fixed;
 }
 
-

```

Comment:
My screens dont really go this small of  a width anyway, I may need to tweak my website to make usable for phone screen sizes